test/e2e/forceRender.test.ts

```diff
@@ -0,0 +1,55 @@
+import * as mwoffliner from '../../src/mwoffliner.lib.js'
```

Comment:
I decided to keep this test as this is the separate testing of `forceRender` param with bm.wikipedia.org